addons/udes_stock/views/stock_picking.xml

```diff
@@ -87,7 +87,7 @@
 
             <!-- Destination location field changed to read-only for goods-in -->
             <xpath expr="//field[@name='location_dest_id']" position="attributes">
-                <attribute name="attrs">{'readonly': [('picking_type_code', '=', 'incoming')]}</attribute>
+                <attribute name="attrs">{'readonly': ['|',('picking_type_code', '=', 'incoming'),('state', 'not in', 'draft')]}</attribute>
```

Comment:
Q: Do we need to consider the destination location on the move lines? This would stop the destination location being changed on the main picking record but not the individual lines.

Reply:
As far as I know, it's not a problem as the individual move lines can be edited but you cannot set them to anything that is not a child of the picking destination location.
But let me know if you think I have missed something

Reply:
I don't think we need to change anything for the move lines as their domain is based on children of the pickings location_dest_id, the problem (as i understand it) is that the location_dest_id could be changed on the pick thus releiving the domain on the move lines, and if the pick is waiting/ready we still want to be able to change what location stuff is being moved to

Reply:
Agree not needed.